AphroCite.h

```diff
@@ -1,59 +1,95 @@
-//
-// Created by Daniel on 14.01.17.
-//
+/*
+    Created by Daniel on 14.01.17.
+*/
 
 #ifndef APHROCITE_APHROCITE_H
 #define APHROCITE_APHROCITE_H
 
-// Includes
+/*
+    Includes
+*/
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdarg.h>
 #include <string.h>
 
-// bool
-typedef unsigned int bool32;
+/*
+    bool
+*/
+typedef unsigned int Boolean32;
```

Comment:
because there is no actual boolean in c its always some form of integer. Threfore Boolean32, but we can also make an enum instead which